### DISCUSSION TITLE
doubt about doc

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#653] [Q&A] [answered]
Hi Lab
what does it mean 'Exactly' EM:Existing Mandatory ?
For example in notice 4 , from https://docs.ted.europa.eu/eforms/latest/schema/parties.html#roleSubrolePresencePerNoticeTable,
Mediation organization is EM but , at least apparently, it is Optional, because if I do not select a TouchPoint or an Organization the notice is still valid.
![image](https://github.com/OP-TED/eForms-SDK/assets/52211037/a6228d2d-9f10-466f-80a6-78d4985fac76)
So there's something I'm missing .. can you explain me ?
K.R
Renato

**Comment by YvesJo** [comment, marked as answer]
Hi,
EM: Existing Mandatory
EM means that when the designated object exists, it is mandatory. There is no technical mean to check that an object exists, so that it is not possible to require the presence of the corresponding field/element which and it appears as optional. Only the Buyer knows whether an object exists or not.
